modules/ROOT/pages/create-community.adoc

```diff
@@ -293,6 +297,8 @@ NOTE: If you ever change the name of your site, be sure to update this setting.
 
 These steps enable guests to register as members in your API Community Manager portal.
 
+Use the following steps to configure custom user settings. To configure users settings automatically with default settings, use the *FIX* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
+
```

Comment:
So, this topic should be a peer to configuring member settings. Again, I think some reorganization and title rewording is required to break up an overly long topic set into more discrete, parallel tasks.

Reply:
Created a Jira for further reorganization in https://www.mulesoft.org/jira/browse/DOCS-11514 .